### ISSUE TITLE
i18n not easy to use in multilanguage project

### ISSUE THREAD

**Issue by jipexu** [ka215/jquery.timeline#2]
Hi
i tried and it's work fine 
```
   $("#myTimeline").timeline({
    i18n : {
    month: { "Jan.": "Janvier", "Fév.": "Février", "Mars": "Mars", "Avr.": "Avril", "Mai": "Mai", "Juin": "Juin", "Juillet": "Juillet", "Août.": "Août", "Sept.": "Septembre", "Oct.": "Octobre", "Nov.": "Novembre", "Déc.": "Décembre" },
    day: { "Lu": "Lundi", "Ma": "Mardi", "Me": "Mercredi", "Jeudi": "Jeudi", "Ve": "Vendredi", "Sa": "Samedi", "Di": "Dimanche" },
    ma: [ "vorm.", "nachm." ]
  },
```

but why not extend the object and use lang files ... with something about like this
`!function(a){a.fn. timeline.i18n.de={month:[  .......`
or
```
(function($) {
      timeline.i18n = $.extend(true,  timeline.i18n, {
        'fr_FR': { ..................
```

 this will be more convienient

**Comment by ka215** [comment]
At the latest v1.0.3, we enhanced the function to support multiple languages. It is a system that retrives the language setting of the browser and loads the language file (JSON format). Please see the README for details.

**Comment by jipexu** [comment]
OH thanks seems very cool !! I read in detail ..

**Comment by jipexu** [comment]
Good good
but why not by  timeline.i18n.us_US ..... timeline.i18n.fr_FR .....  (from the lang pack?) 
The problem is that the browser will always stay in the native language ... if a french want to see in japanese he must not have to change his browser parameter ?...

**Comment by ka215** [comment]
When "timeline.i18n" is extended, the code amount of the plugin option described in the script tag becomes too huge, therefore I thought that maintainability becomes difficult.

To change the import language file, you should change the browser language setting. 
However, by using the newly added "httpLanguage" option (beta), it becomes possible to use the language setting of the browser returned by the server side header.

I know that too if there is no language file 404 error will occur. I am thinking about solving this problem without increasing the inquiries to the server side.

**Comment by jipexu** [comment]
ok ... but do you think we can imagine a simple way to override a var lang ??

**Comment by ka215** [comment]
How to use the httpLanguage option:

* **HttpLanguage: true** - Get the value of "HTTP-Accept-Language" in the response header when accessing the server of the browser and set it as the language to be used. It incorporates the [JQuery-Browser-Language](https://github.com/dansingerman/jQuery-Browser-Language) plugin mechanism.

* **HttpLanguage: false** - Get the default language setting of the browser user agent and set it to the used language (specification implemented in v1.0.3).

The default value of this option is **false**.
Note that with this option true/false, care must be taken as each browser will have different values to be acquired. Since the overhead at the time of true is also large, it is basically recommended to operate with false.

**Comment by ka215** [comment]
Closed because it has already been processed this issue.
